src/dygraph-interaction-model.js

```diff
@@ -405,6 +405,15 @@ DygraphInteraction.endZoom = function(event, g, context) {
   context.dragStartY = null;
 };
 
+// isOutOfExtremes: checks that number p is out of extremes ex.
+// ex is a tuple [a, b], with a <= b
+function isOutOfExtremes(p, ex) {
+
+    return p < ex[0] || p > ex[1]
```

Comment:
please do add explicit semicola at the end of instructions (not just here but everywhere); semicolon insertion is considered harmful